src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
The import of Random is not necessary here, as the randomization should be handled by the FigureSupplier class according to the checklist. Please remove this import and the associated Random object instantiation.